.pubnub.yml

```diff
@@ -1,9 +1,20 @@
 ---
 name: swift
 scm: github.com/pubnub/swift
-version: "5.1.0"
+version: "6.0.0"
 schema: 1
 changelog:
+  - date: 2022-06-01
```

Comment:
@CraigLn this information will be duplicated during bot release run. It should be part of PR description.